content/applications/finance/accounting/reporting/overview/tax_carryover.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Tax carryover
+=============
+
+When performing tax reports, the :guilabel:`tax carryover` feature allows carrying amounts from one
+period to the other without creating new entries.
+
+It has been created to respond to legal requirements in some locations where amounts must be
+transferred from period to period (for example, because the total of the line is negative).
+
+The feature is available per default in countries where it is required, such as Belgium, France, or
+Italy. There is no specific configuration required.
+
+Let’s take an example of a Belgian company that created a credit note of 100 for one of their
+customers. The due tax is 21%.
+
+.. image:: tax_carryover/belgian-example.png
+   :align: center
+   :alt: Illustration with a credit note
+
+In this case, as per local regulation, grid 81 of the tax report may contain a negative amount. But
+it must be declared to the government as zero, and the negative amount should be carried over to the
+next period.
+
+Go to :menuselection:`Accounting --> Reporting --> Tax Report`. You can see that a pop-up on line 81
+explains that the amount will be carried over in the next period.
+
+.. image:: tax_carryover/pop-up.png
+   :align: center
+   :alt: pop-up message stating the amount will be carried over to the next period
```

Comment:
Not sure this last image is compressed , also not sure about the previous ones. As well as not sure if the zoom is 100% --> should be 110%